cve_bin_tool/cli.py

```diff
@@ -524,6 +524,10 @@ def main(argv=None):
             if not source_nvd.nvd_years():
                 with ErrorHandler(mode=error_mode, logger=LOGGER):
                     raise EmptyCache(cvedb_orig.cachedir)
+        if not cvedb_orig.latest_schema():
+            LOGGER.critical("Database does not have the latest schema.")
+            LOGGER.info("Please update database.")
```

Comment:
Maybe add instructions on how to do the database update here?  `--update now` and a link to the docs for offline users?

Reply:
@terriko I agree that we should probably provide the user with some information on how to fix.  @rhythmrx9 I think we should stop the tool as well if we detect we have a mismatch of schemas - if we just issue a warning, the tool will fail when it tries to extract data from the database as some of the columns won't be present.